starter-code/client/src/app/entries.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import { Http } from '@angular/http';
+import 'rxjs/add/operator/map';
+
+@Injectable()
+export class EntriesService {
+
+  BASE_URL: string = 'http://localhost:3000';
+  constructor(private http: Http) {}
+
+  getList() {
+    return this.http.get(`${this.BASE_URL}/api/journal-entries`)
+      .map((res) => res.json());
+  }
+  new(newEntry) {
```

Comment:
don't use new as a name of a method, it can create confusion with the new keyword of the javascript contructor